File: examples/java/src/main/java/MyComponent.java
package javaexample;

import io.github.phenax.h.Component;
import io.github.phenax.h.Layout;
import io.github.phenax.h.node.DOMNode;
import io.github.phenax.h.layouts.EmptyLayout;

public class MyComponent extends Component {

	public Layout getLayout() {
		return new EmptyLayout();
	}

	public DOMNode render() {
		return p(null, "This is text boeey");
	}
}

File: examples/kotlin/src/main/kotlin/MyComponent.kt
package kotlinexample

import io.github.phenax.h.Component
import io.github.phenax.h.Layout
import io.github.phenax.h.node.DOMNode
import io.github.phenax.h.layouts.EmptyLayout

class MyComponent: Component() {

	override val layout = EmptyLayout()

	override fun render(): DOMNode {
		return p(null, "This is text boeey")
	}
}
